### ISSUE TITLE
[addon-info] Filter out props by name

### ISSUE THREAD

**Issue by mindtricksdev** [storybooks/storybook#3408]
I was wondering if there's a way to avoid rendering some props when using `withInfo` from `@storybook/addon-info`.

Right now I'm using the following options to avoid rendering HoC props and it works well:
 ```
{
  propTables: [Icon],
  propTablesExclude: [IconWithStyles],
}
```

But the `Icon` has a `classes: PropTypes.object.isRequired` which gets rendered in the docs and I want to filter this out because it is injected by the HoC. The user does not need to pass this when using the default export `IconWithStyles` component.

Thanks!

**Comment by andyo729** [comment]
I could look into this and try to throw something together.

**Comment by Hypnosphi** [comment]
Released as [`4.0.0-alpha.4`](https://github.com/storybooks/storybook/releases/tag/v4.0.0-alpha.4)
